src/Discord.Net.Rest/API/Rest/ModifyInteractionResponseParams.cs

```diff
@@ -18,5 +18,8 @@ internal class ModifyInteractionResponseParams
 
         [JsonProperty("flags")]
         public Optional<MessageFlags?> Flags { get; set; }
+
+        [JsonProperty("attachments")]
+        public Optional<IAttachment[]> Attatchments { get; set; }
```

Comment:
It should be "Attachments" and not "Attatchments"

Reply:
This has since been removed, you are referencing an outdated commit.

Reply:
Ah, right.

Reply:
Ah, right. I didn't know that.

Reply:
> Ah, right. I didn't know that.

Its all good, appreciate you noticing anyway, I know I didn't until hours later.